streaming/rrdpush.c

```diff
@@ -109,6 +109,9 @@ int rrdpush_init() {
     }
 
     char *invalid_certificate = appconfig_get(&stream_config, CONFIG_SECTION_STREAM, "ssl skip certificate verification", "no");
+    if (!strcmp(invalid_certificate, "no"))
```

Comment:
Why does `"ssl skip certificate verification"` take priority over `"tls skip certificate verification"`?

Reply:
The same argument used for the two previous. This is also used only during the initialization.